client/cypress/integration/login.spec.js

```diff
@@ -0,0 +1,21 @@
+/// <reference types="cypress" />
+
+context('Login', () => {
```

Comment:
Shouldn't this be in a `describe` and not a `context`?

Reply:
hmmm... I know in Karma this is typically describe. This was a copy paste from their example file, and it included context as the keyword. However I can investigate into this one to confirm that. As is, It certainly runs.

Reply:
I think it really depends on what the test runner is expecting. I have mostly used mocha and things built off of it, so I was expecting a `describe`. So long as it works!

Reply:
Yeah. so reading through their documentation. here's what I found:
```
The test interface, borrowed from Mocha, provides describe(), context(), it() and specify().

context() is identical to describe() and specify() is identical to it(), so choose whatever terminology works best for you.
```

However, it does seem odd in the text example that it does a mix and match. `context` and `it`. I'll probably change it in this case to be `describe` then to match the `it` block used in mocha.